packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -350,6 +351,11 @@ class Draggable<T extends Object> extends StatefulWidget {
   /// Defaults to false.
   final bool rootOverlay;
 
+  /// How to behave during hit test.
+  ///
+  /// If null, defaults to [HitTestBehavior.deferToChild].
```

Comment:
Why not just make this non-nullable and default it to deferToChild in the constructor?

Reply:
Yeah, that would be better.